### ISSUE TITLE
CVE - Update Python and Django

### ISSUE THREAD

**Issue by johanxnilsson** [OasisLMF/OasisPlatform#493]
Seems that api_server:1.15.7 (and possibly 1.16.x) contains a few security issues in:
python 3.8.7
django 3.0.11
(also jsonpickle-1.4.2 but there is no fix for this available)

Both can be solved by a minor update to:
python 3.8.8 (or 3.9.2)
django 3.0.12 (or 3.1.6)

![image](https://user-images.githubusercontent.com/64204808/116255688-c77e8500-a772-11eb-8545-7f0058fa4da1.png)

**Comment by sambles** [comment]
- [ ] 1.15.x backports 
- [ ] 1.17.0 release

**Comment by sambles** [comment]
**Note:** CVE has been fixed in `1.16.0`
```
$ docker run --entrypoint /bin/bash -it coreoasis/api_server:1.16.0

oasis-api(36b8f2e7c179) /var/www/oasis # python --version
Python 3.8.9

oasis-api(36b8f2e7c179) /var/www/oasis # pip freeze | grep Django
Django==3.1.7
```
